core/src/test/java/kafka/test/ClusterTestExtensionsTest.java

```diff
@@ -170,12 +185,12 @@ public void testClusterTests() throws ExecutionException, InterruptedException {
         @ClusterTest(types = {Type.ZK, Type.KRAFT, Type.CO_KRAFT}, disksPerBroker = 2),
     })
     public void testClusterTestWithDisksPerBroker() throws ExecutionException, InterruptedException {
-        Admin admin = clusterInstance.createAdminClient();
```

Comment:
Currently, all admins created by `clusterInstance.createAdminClient` are stored in a collection. They will get closed after tests, so we don't need to close they manually. I know that is a bit weird to IDE, but that is a style in code base so we can keep it as it is

Reply:
Changed. Thanks!